docs/supported-technologies.asciidoc

```diff
@@ -117,6 +117,12 @@ The agent supports <<framework-integrations,integrations with certain frameworks
 To instrument custom metrics, like rendering time or mounting time of components on frameworks like React, Angular, Vue,
 etc., use the <<custom-transactions,custom transactions API>>.
 
+The core RUM agent supports MPAs and SPAs out of the box. This means that for SPAs to use <<framework-integrations,framework specific integrations>> is a recommendation rather than a requirement. We recommend them for the following reasons:
+
+* They group pages by routes better. This results in `transaction.name` mapped to the actual application route path the users have declared in their SPA application.
+* They exactly know when a SPA navigation is rendered, the core RUM agent does not know that and waits for the last network request before considering the SPA transition is finished. The integration packages on the other hand hook into the component lifecycle and measures the duration of the SPA transition correctly.
```

Comment:
nit typo, change "...when a SPA navigation..." to "...when an SPA navigation..."